<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaporDataToolDialogBase</class>
 <widget class="QDialog" name="WaporDataToolDialogBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>689</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>528</width>
    <height>689</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>528</width>
    <height>689</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WaPOR-Data Tool</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>660</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tab_pages">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>511</width>
     <height>641</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>Data Extraction</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_data_extents">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>481</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Select Extent</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <widget class="QgsMapLayerComboBox" name="mMapLayerComboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>291</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>34</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Layer</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chb_clip_to_cutline">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Clip to Cutline</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_retrieve_catalog">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>440</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Catalog</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>480</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Retrieve Data</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_download_options">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>331</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Set Download Options</string>
     </property>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>239</width>
         <height>39</height>
        </rect>
       </property>
       <widget class="QDateEdit" name="date_from">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="date">
         <date>
          <year>2000</year>
          <month>1</month>
          <day>3</day>
         </date>
        </property>
       </widget>
       <widget class="QDateEdit" name="date_to">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>118</x>
          <y>13</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Period</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="txb_download_location">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>271</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_set_download_location">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>31</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Download Directory</string>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>570</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>481</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Analysis</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>481</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Authentication</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_token_status">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Token Status</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_update_token">
      <property name="geometry">
       <rect>
        <x>384</x>
        <y>62</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Token</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="wapor_tokenbox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>63</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_check_token">
      <property name="geometry">
       <rect>
        <x>226</x>
        <y>26</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Re-Check</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>149</y>
       <width>481</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Defaults</string>
     </property>
     <widget class="QPlainTextEdit" name="txt_default_dir_path">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_browse_default_download_dir">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>60</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>351</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Default Download Directory</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>WaporDataToolDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>WaporDataToolDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
